content/plugins/gearsdigital/enhanced-toolbar-link-dialog/plugin.txt

```diff
@@ -14,4 +14,4 @@ Subcategory: enhancement
 
 ----
 
-Description: The Enhanced Toolbar Link Dialog makes it easy to add internal or external references to your content by extending the default link dialog of Kirby's textarea fields
+Description: Extend Kirby's default link dialog with a search functionality to create links to existing or external pages right at your fingertips.
```

Comment:
I like the rephrasing except that now it doesn't mention anymore that this is in context of the textarea field. Would be good to make that relation.

Reply:
@distantnative The point of the new version is that it is no longer limited to the textarea field.

Reply:
Ok, got it :D

Reply:
@texnixe, correct, that was my intention. Thanks for accepting this MR!